blog/retention analysis.md

```diff
@@ -0,0 +1,31 @@
+
+In the first few months, the retention rate of A is lower than that of B, and the simple conclusion is that APP A is not as good as APP B. Conversely, if we take a long-term view, we find that the retention rate of APP A continues to decline, while the retention rate of APP B slowly converges to a certain value. The retention trend of an excellent product should have the following characteristics:
+
+- the retention rate in the horizontal view should stay at a fixed value. For example, there are 100 inexperienced users in a certain month and the retention rate is stable at 50% after half a year, which means that this group of users will be valuable for the company. Otherwise, even though the retention rate declines very slowly, it may go to zero in the end. No matter how many inexperienced users there are, it is meaningless for the long-term development.
+- the retention data in the vertical view should be getting better. The company should constantly improve products and experiences based on users’ feedback. The users who join later should enjoy a more excellent product and service, thus leading to a higher retention rate.
+  
+But if we think a little bit more carefully. It is found that the promotion of APP A is mostly online and A small part offline, while APP b is just the opposite. Consequently, if we want to analyze the data more precisely, we need to break it down a bit more.
+
+### Behavioral cohorts:
+Another popular method to group the users is based on the behaviors. For example, after we obtain user data after they have registered the APP A, we can further divide the cohort based on the usage time or their preferred function when using this APP. Then, the columns of the table contain the cohort which is determined by the behavior, and the row data represents the observed activity in the APP. Compared to acquisition cohort, behavioral cohort depends more on the characteristics of the product and should be able to capture more valuable user information if it is utilized effectively. As Kelvin said, “If you cannot measure it, you cannot improve it.”
```

Comment:
1. please ref to the retention figure.
2. please add more details about this part. A practical example may be better to illustrate the problem.